Domain.Supply/Entities/Supply/MedicalSupplyEntity.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Diagnosea.Submarine.Abstractions.Enums;
+
+namespace Diagnosea.Submarine.Domain.Supply.Entities.Supply
+{
+    public class MedicalSupplyEntity : SupplyEntity
```

Comment:
You're going to need to include a `Dosage` or `ReccomendedDosage` in some format.